.env.local

```diff
@@ -19,11 +19,11 @@ MAILGUN_API_KEY:                        key
 MAILGUN_DOMAIN:                         sandbox43bf51fb258844f0bc393e806ebd25bb.mailgun.org
 FROM_EMAIL_ADDRESS:                     info@u-can-act.nl
 
-AUTH0_CLIENT_ID:                        'MoWP7FzVx8fIELYmsPcY8NS4FYV_auZX'
-AUTH0_DOMAIN:                           'u-can-act-dev.eu.auth0.com'
-AUTH0_CLIENT_SECRET:                    'Im8alnb8wsEKdZyJrdjqd3YzONIPuHD_vxflz-oYedvZy9eFFe4SzrzGtu-EwWCx'
-AUTH0_REDIRECT_URL:                     'http://vsv.io/admin/callback'
-AUTH0_AUDIENCE:                         'https://u-can-act-dev.eu.auth0.com/api/v2/'
+AUTH0_CLIENT_ID=MoWP7FzVx8fIELYmsPcY8NS4FYV_auZX
+AUTH0_DOMAIN=u-can-act-dev.eu.auth0.com
+AUTH0_CLIENT_SECRET=Im8alnb8wsEKdZyJrdjqd3YzONIPuHD_vxflz-oYedvZy9eFFe4SzrzGtu-EwWCx'
```

Comment:
er staan hier quotjes achter, dat lijkt me niet helemaal goed. Misschien overal `=` gebruiken?

Reply:
Nou ik wilde eigenlijk alleen de variabelen exposen naar JS die ook echt in JS gebruikt worden. Dan houden we goed het onderscheid. 
dubbele punt is env vars die in ruby gebruikt worden,
equals sign is env vars die (ook) in JS gebruikt worden.

Maar idd die single quote aan het eind hoort daar niet ik haal m weg.